popup.html

```diff
@@ -32,7 +32,7 @@
               target="_blank"
               href="https://github.com/AlkTheOrg/imdb-watchlist-randomizer/blob/main/ABOUT.md#about-the-extension"
               >About</a
-            >
+			>
```

Comment:
Unneeded blank

Reply:
noticed

Reply:
in my text editor, this spacing doesn't exit